src/Server/GenericServer.php

```diff
@@ -15,6 +15,86 @@
  */
 namespace League\OAuth1\Client\Server;
 
+use League\OAuth1\Client\Credentials\TokenCredentials;
+use Psr\Http\Message\ResponseInterface;
+
 class GenericServer extends AbstractServer
 {
+    /**
+     * Temporary credentials uri.
+     *
+     * @var string
+     */
+    protected $temporaryCredentialsUri;
+
+    /**
+     * Authorization uri.
+     *
+     * @var string
+     */
+    protected $authorizationUri;
+
+    /**
+     * Token credentials uri.
+     *
+     * @var string
+     */
+    protected $tokenCredentialsUri;
+
+    /**
+     * Resource owner details uri.
+     *
+     * @var string
+     */
+    protected $resourceOwnerDetailsUri;
+
+    /**
+     * {@inheritdoc}
```

Comment:
Good catch here too. I think these have been hanging around this project for some time.